ads/google/doubleclick.md

```diff
@@ -16,116 +16,4 @@ limitations under the License.
 
 # Doubleclick
 
-## Example
```

Comment:
Why was this content removed?  How will publishers know what to put in their "amp-ad" tag when they insert it on the page.  The "new" documentation doesn't provide as clear guidance and assistance on adding "amp-ad" tags. I was actually looking for a Doubleclick example and had to dig into "Fast Fetch" implementation docs, which is lower-level information that I consider too much for publishers.  Is there supplemental documentation coming?


cc: @jasti